src/loader.js

```diff
@@ -18,6 +18,7 @@ const reactModule = /['"]react['"]/;
 function RefreshHotLoader(source, inputSourceMap) {
   // Add dependency to allow caching and invalidations
   this.addDependency(path.resolve('./runtime/RefreshModuleRuntime'));
+  this.cacheable();
```

Comment:
Ah, I thought loader results are by default Cacheable?

I assume the parameter passed in is true, will `this.cacheable(true);` be more explicit?

Also - please add a comment so that we know what's that line for.

Reply:
I think webpack docs are confusing. The *argument* is `true` by default, but if you don't call it, it won't be cacheable.

Reply:
I agree that it is a bit confusing ... For safety I'll find some time to dig into the actual loader call and see how Cacheable is being called.

Reply:
This is weird. By default the `cacheable` flag of the loader is indeed set as true, so I think something is busting the cache.